handbook/direction/code-graph/index.md

```diff
@@ -0,0 +1,56 @@
+# Code Graph
+
+The Code Graph org’s builds and scales the Code Graph and how our users interact with it. We are responsible for search (the user experience as well as performance), insights, code intelligence, and batch changes. This page describes overarching initiatives that tie multiple teams and milestones within Code Graph together.
+
+Individual team direction pages within Code Graph can be found on our [direction home page](../index.md#code-graph). For details on how the themes here become reality, as well as what each team is working on next for their individual visions, take a look there.
+
+## Mission
+
+What is the team's reason for existing? What is the fundamental value they aim to provide? Who are your key customers/stakeholders and why, and are there any customers/stakeholders that you're explicitly choosing not to address that people might have expected you to?
+
+## Vision
+
+### 1 Year vision
+
+> Add a sentence or two here indicating what the most important things you want to achieve/problems you want to solve in the next year are. These should in most cases move your three and ten year visions forward.
```

Comment:
I think once we complete the themes, the one year vision will come to us naturally as the culmination of delivering those.